app/controllers/dashboards_controller.rb

```diff
@@ -29,6 +29,7 @@ def show
     @articles = @articles.sorting(params[:sort]).decorate
     @articles = Kaminari.paginate_array(@articles).page(params[:page]).per(50)
 
+    @collections_count = @user.collections.count
```

Comment:
**question (non-blocking):** Is `@collections_count` used anywhere? The view here uses `@user.collections.count` too.

Reply:
This was an oversight, I have removed it.